sql/src/test/scala/org/datasyslab/geosparksql/adapterTestScala.scala

```diff
@@ -132,6 +133,12 @@ class adapterTestScala extends TestBaseScala {
       assert (df.columns(1) == "STATEFP")
     }
 
+    it("Read GeoJson that contains id"){
+      val rdd = GeoJsonReader.readToGeometryRDD(sparkSession.sparkContext, "sql/src/test/resources/idContainsGeoJson.json", false, false)
```

Comment:
Please follow the coding style in other test cases and use the proper way to read the input file path